erts/doc/src/erlang.xml

```diff
@@ -52,6 +59,300 @@
+      <name name="nonempty_list" n_vars="0"/>
+      <desc>
+        <p>A <seetype marker="#list">list()</seetype> that contains some items.</p>
+      </desc>
+    </datatype>
+    <datatype>
+      <name name="nonempty_list" n_vars="1"/>
+      <desc>
+        <p>A <seetype marker="#list">list(ContentType)</seetype> that contains some items.</p>
+      </desc>
+    </datatype>
+    <datatype>
+      <name name="nonempty_maybe_improper_list" n_vars="0"/>
+      <desc>
+        <p>A <seetype marker="#maybe_improper_list">maybe_improper_list()</seetype> that contains some items.</p>
+      </desc>
+    </datatype>
+    <datatype>
+      <name name="nonempty_maybe_improper_list" n_vars="2"/>
+      <desc>
+        <p>A <seetype marker="#maybe_improper_list">maybe_improper_list(ContentType, TerminationType)</seetype>
+          that contains some items.</p>
+      </desc>
+    </datatype>
+    <datatype>
+      <name name="nonempty_string" n_vars="0"/>
+      <desc>
+        <p>A <seetype marker="#string">string()</seetype> that contains some characters.</p>
+      </desc>
+    </datatype>
+    <datatype>
+      <name name="number" n_vars="0"/>
+      <desc>
+        <p>An Erlang <seeguide marker="system/reference_manual:data_types#number">number</seeguide>.</p>
+      </desc>
+    </datatype>
+    <datatype>
+      <name name="pid" n_vars="0"/>
+      <desc>
+        <p>An Erlang <seeguide marker="system/reference_manual:data_types#pid">process identifier</seeguide></p>
+      </desc>
+    </datatype>
+    <datatype>
+      <name name="port" n_vars="0"/>
+      <desc>
+        <p>An Erlang <seeguide marker="system/reference_manual:data_types#port-identifier">port identifier</seeguide></p>
+      </desc>
+    </datatype>
+    <datatype>
+      <name name="pos_integer" n_vars="0"/>
+      <desc>
+        <p>A positive integer.</p>
+      </desc>
+    </datatype>
+    <datatype>
+      <name name="reference" n_vars="0"/>
+      <desc>
+        <p>An Erlang <seeguide marker="system/reference_manual:data_types#reference">reference</seeguide>.</p>
+      </desc>
+    </datatype>
+    <datatype>
+      <name name="string" n_vars="0"/>
+      <desc>
+        <p>A character string represented by a list of ASCII characters or unicode codepoints.</p>
+      </desc>
+    </datatype>
+    <datatype>
+      <name name="term" n_vars="0"/>
+      <desc>
+        <p>A piece of data of any data type.</p>
+      </desc>
+    </datatype>
+    <datatype>
+      <name name="timeout" n_vars="0"/>
+      <desc>
+        <p>A timeout value that can be passed to a <seeguide marker="system/reference_manual:expressions#receive">receive expression</seeguide>.</p>
+      </desc>
+    </datatype>
+    <datatype>
+      <name name="tuple" n_vars="0"/>
+      <desc>
+        <p>An Erlang <seeguide marker="system/reference_manual:data_types#tuple">tuple</seeguide>.</p>
+      </desc>
+    </datatype>
+    
+    <datatype_title>Not-Predefined datatypes</datatype_title>
```

Comment:
Perhaps you should an explanation to explain that these data types are not part of the language and will need an `erlang` prefix.

```suggestion

    <datatype_title>Other Datatypes</datatype_title>
```

Reply:
> Perhaps you should an explanation to explain that these data types are not part of the language and will need an `erlang` prefix.

I would need to expand the dtd then as it does not allow me to do that as it is now. I may take try to make it so that you can write `erlang:ext_binary` and then the prefix will be there, just as we do with the rest of the functions in the module.